bootstrap/dev/venv.sh

```diff
@@ -11,3 +11,5 @@ export VENV_ARGS="--python python2"
   -e letsencrypt-nginx \
   -e letshelp-letsencrypt \
   -e letsencrypt-compatibility-test
+
+find . -iname '*.pyc' -exec rm '{}' ';'
```

Comment:
The previous command would error, causing the script to exit due to -e.  But I can double-mitigate this.